<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSettingsWidget</class>
 <widget class="QWidget" name="GameSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="1">
      <widget class="QComboBox" name="m_variantCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_variantLabel">
       <property name="text">
        <string>Variant:</string>
       </property>
       <property name="buddy">
        <cstring>m_variantCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="m_timeControlBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_timeControlLabel">
       <property name="text">
        <string>Time Control:</string>
       </property>
       <property name="buddy">
        <cstring>m_timeControlBtn</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Opening suite</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="m_pgnEpdLabel">
        <property name="text">
         <string>PGN/EPD file:</string>
        </property>
        <property name="buddy">
         <cstring>m_openingSuiteEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="m_openingSuiteEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_browseOpeningSuiteBtn">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_openingOrderLabel">
        <property name="text">
         <string>Opening order:</string>
        </property>
        <property name="buddy">
         <cstring>m_seqOrderRadio</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="m_seqOrderRadio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Sequential</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_randomOrderRadio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_openingSuiteDepth">
        <property name="text">
         <string>Depth:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="m_openingSuiteDepthSpin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> plies</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_fenLabel">
        <property name="text">
         <string>FEN string:</string>
        </property>
        <property name="buddy">
         <cstring>m_fenEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_fenEdit">
        <property name="maxLength">
         <number>128</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_openingBookGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Opening book</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_polyglotFileLabel">
        <property name="text">
         <string>Polyglot file:</string>
        </property>
        <property name="buddy">
         <cstring>m_polyglotFileEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="m_polyglotFileEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_browsePolyglotFile">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_polyglotDepthLabel">
        <property name="text">
         <string>Depth:</string>
        </property>
        <property name="buddy">
         <cstring>m_polyglotDepthSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_polyglotDepthSpin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> full moves</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_polyglotAccessMode">
        <property name="text">
         <string>Access mode:</string>
        </property>
        <property name="buddy">
         <cstring>m_ramAccessRadio</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="m_ramAccessRadio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The whole book is loaded in memory. This makes book access a bit faster but uses more memory.</string>
          </property>
          <property name="text">
           <string>Ram</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_diskAccessRadio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The book is accessed directly from disk. This is a good choice for very large books.</string>
          </property>
          <property name="text">
           <string>Disk</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_6">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QGroupBox" name="m_drawAdjudicationGroup">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Draw adjudication</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="m_drawMoveNumberLabel">
          <property name="text">
           <string>Move number:</string>
          </property>
          <property name="buddy">
           <cstring>m_drawMoveNumberSpin</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_drawMoveNumberSpin">
          <property name="toolTip">
           <string>The game can be adjudicated as a draw after this many full moves have been played. A value of 0 disables draw adjudication.</string>
          </property>
          <property name="specialValueText">
           <string>Off</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_drawMoveCountLabel">
          <property name="text">
           <string>Move count:</string>
          </property>
          <property name="buddy">
           <cstring>m_drawMoveCountSpin</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="m_drawMoveCountSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The scores of both players need to be within the chosen threshold for this many consecutive moves.</string>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="m_drawScoreLabel">
          <property name="text">
           <string>Score:</string>
          </property>
          <property name="buddy">
           <cstring>m_drawScoreSpin</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="m_drawScoreSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Both players' scores must be within this many centipawns from zero.</string>
          </property>
          <property name="suffix">
           <string> cp</string>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="m_resignAdjudicationGroup">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Resign adjudication</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="m_resignMoveCountLabel">
          <property name="text">
           <string>Move count:</string>
          </property>
          <property name="buddy">
           <cstring>m_resignMoveCountSpin</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_resignMoveCountSpin">
          <property name="toolTip">
           <string>Resign adjudication is used if an engine's score is below the threshold for at least this many consecutive moves.</string>
          </property>
          <property name="specialValueText">
           <string>Off</string>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_resignScoreLabel">
          <property name="text">
           <string>Score:</string>
          </property>
          <property name="buddy">
           <cstring>m_resignScoreSpin</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="m_resignScoreSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The score must be at least this many centipawns below zero.</string>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string> cp</string>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="m_gameLengthGroup">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Game Length</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_maxGameLengthLabel">
          <property name="text">
           <string>Limit:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_maxGameLengthSpin">
          <property name="toolTip">
           <string>The game will be adjucated as drawn when move limit is reached</string>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="specialValueText">
           <string>Off</string>
          </property>
          <property name="suffix">
           <string> moves</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_tbCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Tablebase adjudication</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_ponderingCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Thinking on opponent's time</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
